flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CollectionFunctionsITCase.java

```diff
@@ -287,4 +301,46 @@ Stream<TestSetSpec> getTestSetSpecs() {
                                 "Invalid input arguments. Expected signatures are:\n"
                                         + "ARRAY_REMOVE(haystack <ARRAY>, needle <ARRAY ELEMENT>)"));
     }
+
+    private Stream<TestSetSpec> arrayReverseTestCases() {
+        return Stream.of(
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.ARRAY_REVERSE)
+                        .onFieldsWithData(
+                                new Integer[] {1, 2, 2, null},
+                                null,
+                                new Row[] {
+                                    Row.of(true, LocalDate.of(2022, 4, 20)),
+                                    Row.of(true, LocalDate.of(1990, 10, 14)),
+                                    Row.of(true, LocalDate.of(1990, 10, 14)),
+                                    Row.of(true, LocalDate.of(1990, 10, 14)),
```

Comment:
should we have all these 3 lines  in the code?
It seems we cannot differentiate them... So seems not much value

Reply:
fixed